### PR TITLE
Use the english version of the task's files and dirs

### PR DESCRIPTION
**Pull request by edomora97**
I've also rewritten the `new_contest` in python since the bash version was OS dependent (date vs gdate shenanigans) and very cryptic.

New folder structure:

```
round1
├── contest.yaml
└── test
    ├── att
    │   ├── input0.txt -> ../statement/test.input0.txt
    │   ├── input1.txt -> ../statement/test.input1.txt
    │   ├── output0.txt -> ../statement/test.output0.txt
    │   ├── output1.txt -> ../statement/test.output1.txt
    │   ├── test.c
    │   ├── test.cpp
    │   ├── test.java
    │   ├── test.pas
    │   └── test.py
    ├── check
    │   └── checker.cpp
    ├── gen
    │   ├── GEN
    │   ├── generator.py
    │   ├── limiti.py
    │   └── validator.py
    ├── sol
    │   ├── soluzione.cpp
    │   ├── soluzione_py.py
    │   ├── template_c.c -> ../att/test.c
    │   ├── template_cpp.cpp -> ../att/test.cpp
    │   └── template_py.py -> ../att/test.py
    ├── statement
    │   ├── english.pdf
    │   ├── english.tex
    │   ├── logo.pdf -> ../../../tools/new_task/logo.pdf
    │   ├── statement.pdf -> english.pdf
    │   ├── test.input0.txt
    │   ├── test.input1.txt
    │   ├── test.output0.txt
    │   └── test.output1.txt
    └── task.yaml
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,23 +28,23 @@ The main folder only contains the single file `task.yaml`, with general setup in
...
-### `cor` folder
+### `check` folder
 
-This folder contains a single source file of a checker, that attributes a score (from 0 to 1) to a test output, knowing the test input and the correct test output. Many tasks do not need a checker, as plain white-diff scoring is sufficient: in that case, the folder can be fully omitted. For the remaining tasks, the skeleton includes a general-purpose checker with safe read operations that can be modified to fit various needs. 
+This folder contains a single source file of a checker, that attributes a score (from 0 to 1) to a test output, knowing the test input and the correct test output. Many tasks do not need a checker, as plain white-diff scoring is sufficient: in that case, the folder can be fully omitted. For the remaining tasks, the skeleton includes a general-purpose checker with safe read operations that can be modified to fit various needs.
 
 ### `gen` folder
 
 This folder contains the input generation data, split into few files:
 - `limiti.py` with a description of the limits of the various subtasks;
```
Comment (Harniver) [inline README.md:45]: translate this to limits.py as well?

File: new_task/validator.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env pypy3
+#!/usr/bin/env python3
```
Comment (Harniver) [inline new_task/validator.py:1]: pypy3 is probably going to validate faster, but I guess it isn't crucial

File: README.md (hunk 1 above)
Comment (Harniver) [inline README.md:45]: translate to limits.py, at this point

**Review by Harniver — CHANGES_REQUESTED** [review]

File: new_task/validator.py (hunk 2 above)
Comment (edomora97) [reply new_task/validator.py:1]: Yes, and at the moment pypy3 is quite broken on archlinux.

**Pull request merged**